Library/Formula/wdt.rb

```diff
@@ -0,0 +1,74 @@
+class Wdt < Formula
+  desc "Transfer data as fast as possible over multiple TCP paths."
+  homepage "https://github.com/facebook/wdt"
+
+  stable do
+    url "https://github.com/facebook/wdt/archive/v1.26.1603040.tar.gz"
+    sha256 "b0fffdd12478ef74de50852b9c356352dae5f7ccee22dbca91a88afc6c5b6fee"
+
+    resource "folly" do
+      url "https://github.com/facebook/folly/archive/v0.57.0.tar.gz"
+      sha256 "92fc421e5ea4283e3c515d6062cb1b7ef21965621544f4f85a2251455e034e4b"
+    end
+
+    patch :DATA
```

Comment:
In fact, the issue only occur when we use wdt v1.26 and folly 0.57, because the folly git head has removed the Malloc.cpp, and wdt BUILD.md also suggest to use folly's git head, if you build with --HEAD, the patch will be unnecessary. 

So, I doubt the patch will not be accept, but I'm not sure whether homebrew could mix the stable and git head in one build?